oak-run/src/main/js/oak-mongo.js

```diff
@@ -352,12 +352,12 @@ var oak = (function(global){
      * @param {string} pattern the pattern to match the nodes to be removed.
      */
     api.removeDescendantsAndSelfMatching = function(pattern) {
-        var count = 0;
+        let count = 0;
```

Comment:
It seems like this change is a no-op? What would make it still useful to have?

Reply:
the variables defined with `let` are block scoped and hence cause less confusion and are less error-prone. But I agree this can be reverted.

Will remove this.

Reply:
> the variables defined with `let` are block scoped and hence cause less confusion and are less error-prone. But I agree this can be reverted.
> 
> Will remove this.

Removed in https://github.com/apache/jackrabbit-oak/pull/1477/commits/30672ee37040d8d280ade40dddc35a7bab5063f5